src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/lines/TopLine.java

```diff
@@ -0,0 +1,101 @@
+package org.betonquest.betonquest.compatibility.holographicdisplays.lines;
+
+import lombok.Getter;
+import me.filoghost.holographicdisplays.api.hologram.Hologram;
+import org.betonquest.betonquest.compatibility.holographicdisplays.TopXLine;
+import org.betonquest.betonquest.compatibility.holographicdisplays.TopXObject;
+
+import java.util.Arrays;
+
+/**
+ * Creates a new instance for TopLine.
+ */
+@SuppressWarnings("PMD.CommentRequired")
```

Comment:
New Classes should not have any `@SuppressWarnings`, except false positives

And `CommentRequired` is not allowed at all for new Classes.

Reply:
Just to be sure, what exactly should I include in the comment? I haven't found such comments in the latest commits. I thought of `@since` and maybe `@author`?

Reply:
In general you try to describe what a field is for or what it exactly stores. For methods you try to describe the input, output  and processing behaviour. Simply try a compromise of adding it without spending to much time for it

Reply:
Oh yeah I see, I am not quite sure why I even added them in the first place. When I removed them everything worked perfectly fine. I thought the error resulted from the missing tags I mentioned.